Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -290,6 +289,15 @@ class SearchSettingsTableViewController: UITableViewController {
             }
         }
     }
+    
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        if let engine = customSearchEngines[safe: indexPath.row],
+           engine == searchEngines.defaultEngine(forType: .standard) {
```

Comment:
Should we make a check for `privateBrowsing` mode too?
What if user sets it for priv mode

Reply:
They cant set it for Private mode, it was one of the security requirements so we disabled.
https://github.com/brave/brave-ios/pull/3350 you can see the change here.